From 1284c176086f6e4ef0cc0f025456ea3113f02e3d Mon Sep 17 00:00:00 2001
From: Russell Bryant <rbryant@redhat.com>
Date: Wed, 14 Mar 2012 14:14:39 -0400
Subject: [PATCH] Avoid "No handlers could be found for ..." message.

Fix bug 936404.

Avoid "no handlers could be found for keystoneclient.client" messages.
This log message doesn't really seem necessary, so I just removed it.

Change-Id: Ic9a608346b7d5f794d6022672e9a295b00472fc5
---
 keystoneclient/client.py |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/keystoneclient/client.py b/keystoneclient/client.py
index 3042d09..73e7f9a 100644
--- a/keystoneclient/client.py
+++ b/keystoneclient/client.py
@@ -126,7 +126,6 @@ class HTTPClient(httplib2.Http):
             body = None
 
         if resp.status in (400, 401, 403, 404, 408, 409, 413, 500, 501):
-            _logger.exception("Request returned failure status.")
             raise exceptions.from_response(resp, body)
         elif resp.status in (301, 302, 305):
             # Redirected. Reissue the request to the new location.
-- 
1.7.7.6

